### ISSUE TITLE
cannot find the version 1.0.30001228?

### ISSUE THREAD

**Issue by qiutian00** [browserslist/caniuse-lite#77]
When i use `npx browserlist@latest --update-db` command, install caniuse-lite version is 1.0.30001228 ?

![image](https://user-images.githubusercontent.com/24382500/118747643-fb8a2900-b88c-11eb-945f-760e73e3b7d2.png)

**Comment by ai** [comment]
Yeap `1.0.30001228` is the latest version of `caniuse-lite` on npm https://www.npmjs.com/package/caniuse-lite

What is the problem with this output?

**Comment by qiutian00** [comment]
It's my fault, just before I not found it(v1.0.30001228 ) in the version tag.

**Comment by ai** [comment]
Yeap. We recently stopped to use tags.

**Comment by qiutian00** [comment]
Ooh.

**Comment by clemens** [comment]
@ai May I ask why? It's incredibly helpful to have tags for various reasons.

**Comment by ai** [comment]
@clemens I did not see use cases. Tell me why do you need them.

**Comment by clemens** [comment]
@ai Well, the three main reasons I like tags:

- Software packages change licenses – it doesn't happen often, but it does happen and sometimes with a huge blast radius (e.g. https://www.theregister.com/2021/03/29/rails_mime_fix/). I use [LicenseFinder](https://github.com/pivotal/LicenseFinder) to manage the licenses we use and I put the links to licenses in the documentation. If I'd link to the main branch, the license info might change and be out of sync with the version I have installed => not ideal. I could link to the particular commit, but that's tedious. => Tags are a perfect match for this.
- When I have issues that I'm tracing back to a certain package, I like to open the source code and take a look around. Now obviously I want to look at the version that I'm using, not always the status quo of the main branch => again, while I could checkout a specific commit using Git (or even download the NPM package and inspect it), tags are just that much more pleasant to use here.
- It goes together very well with natural things like releases, changelogs etc.

In short: It would be very beneficial to have tags and I don't see any downsides. I'd assume you have some kind of automation that builds your packages and uploads them to NPM => wouldn't that be the perfect place to simply add a `git tag` command incl. a push to GitHub and a release (via the GitHub API)? This way, even if you don't want/need to write changelogs, at least releases (and corresponding tags) are visible.

**Comment by ai** [comment]
Sure. Send PR with tag adding in auto-release script.
